* Version should be the version of the desktop entry specification, not
  application version.
* Application category is deprecated.
--- a/opt/Webex/bin/webex.desktop	2022-07-11 14:27:19.000000000 +0200
+++ b/opt/Webex/bin/webex.desktop	2022-08-05 11:35:53.335000000 +0200
@@ -1,10 +1,9 @@
 [Desktop Entry]
-Version=42.7.0.22904
 Name=Webex
-Comment=Webex
+Comment=Video conferencing
 Exec=/opt/Webex/bin/CiscoCollabHost %U
 Icon=/opt/Webex/bin/sparklogosmall.png
 Terminal=false
 Type=Application
-Categories=Utility;Application;
+Categories=Network;
 MimeType=x-scheme-handler/webexteams;x-scheme-handler/ciscospark;x-scheme-handler/webex
